src/netstandard/ref/System.cs

```diff
@@ -2969,7 +2969,7 @@ public partial class MemberAccessException : System.SystemException
         public MemberAccessException(string message) { }
         public MemberAccessException(string message, System.Exception inner) { }
     }
-    public readonly partial struct Memory<T>
+    public readonly partial struct Memory<T> : System.IEquatable<System.Memory<T>>
     {
         private readonly object _dummy;
         private readonly int _dummyPrimitive;
```

Comment:
This looks fine, but below there are indexer overloads for Range. I thought those were removed.

Reply:
Never mind, I just saw the last PR.